### DISCUSSION TITLE
[Feature Request] Merge RegEx into Folder search or Tag search

### DISCUSSION THREAD

**Discussion by xmbai9507** [OlegLustenko/obsidian-bulk-rename#19] [Ideas]
Thanks for this awesome plugin. Is it possible RegEx to be merged into Folder search or Tag search, so that allowing users using RegEx to batch rename part of files within a specific folder or part of files having the same tag? In this case, leaving the "Folder location" or "Tag names" field blank could conduct a global search.

**Comment by OlegLustenko** [comment]
Yes! Absolutely possible

**Reply by NullAleph** [reply]
This would be awesome - the ability to do RegEx patterns to rename files in a specific folder.  Thanks, Oleg!

**Comment by OlegLustenko** [comment]
So can this flow be covered like this

1) Search by RegExp patterns
2) Replace based on RegExp patterns in Existing and Replacement fields?

read more there: https://github.com/OlegLustenko/obsidian-bulk-rename/releases/tag/0.5.2


It's not a big deal to make it more selective even per directory+RegExp but it might require revisiting existing layout
